### PR TITLE
remove FeedbackPrompt

### PR DESCRIPTION
**Pull request by sqs**
The feedback prompt was shown in the top right in the global navbar:

image
image

While I like the idea of an easy feedback widget, this one is currently worse than nothing because we don't monitor it or have the ability to effectively respond:

- Submissions are anonymous and do not include the username or email address of the user. We can't easily get in touch with them in the current form.
- Submissions are not being responded to. Submissions go to productboard, but it gets lost there, and I don't see any recent responses to these.

It is also somewhat duplicative of the periodic survey that users see every 30 days unless they dismiss it.

History: This was added 2 years ago. It has only received general/maintenance fixes since then (such as accessibility and global refactors).

(If we want to revive it later when we can give it some attention, we can always revert this commit easily.)

## Test plan

Confirmed that the feedback prompt does not show in the global navbar or in the one other page that used it (code insights getting started).

### CODE REVIEW

**Review by valerybugakov — APPROVED** [review]
LGTM ✅

File: client/web/src/enterprise/insights/pages/landing/getting-started/components/code-insights-learn-more/CodeInsightsLearnMore.tsx [hunk 1]
```diff
@@ -62,11 +56,7 @@ export const CodeInsightsLearnMore: React.FunctionComponent<
                         Have a question or idea about Code Insights? We want to hear your feedback!
                     </Text>
 
-                    <FeedbackPrompt onSubmit={handleSubmitFeedback}>
-                        <PopoverTrigger as={Button} variant="link" className={styles.feedbackTrigger}>
-                            Share your thoughts
-                        </PopoverTrigger>
-                    </FeedbackPrompt>
+                    <FeedbackText headerText="" />
```
[the remark this answers is not in the document]
Comment (sqs) [reply client/web/src/enterprise/insights/pages/landing/getting-started/components/code-insights-learn-more/CodeInsightsLearnMore.tsx:58]: Good catch. Actually, I wanted to omit the "Questions/feedback?" text here. I pushed another commit that makes this possible.

**Comment by sqs** [comment]
I did not mean to merge this. I had accidentally set it to auto-merge. Revert PR is in #42397. cc @malomarrec

**Comment by vovakulikov** [comment]
@sqs have you had feedback from @malomarrec that we still need to have this `Feedback` widget in our app? I see that this PR has been merged, but then it was reverted, so I'm curious do we still want to remove this feedback component of it's still needed? 

If it's no longer needed I would be happy to reopen this PR)

**Pull request merged**